rfc6962-bis.xml

```diff
@@ -965,7 +983,7 @@ an accepted root certificate.
             </list></t>
 	  </list>
 
-          The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v1 STH.
+          The <spanx style="verb">hash</spanx> must be calculated as defined in <xref target="tree"/>. The <spanx style="verb">tree_size</spanx> must designate an existing v1 STH. Because of skew, the log server may not know the requested STH. In that case, it will return the latest STH it knows, along with an inclusion proof to that STH. If the server knows the requested STH then only <spanx style="verb">leaf_index</spanx> and <spanx style="verb">audit_path</spanx> are returned.
```

Comment:
We already say that the tree size specifies an STH, so it seems fine to say STH from then on?